kubejs/server_scripts/base/tags/items/forge/terracotta.js

```diff
@@ -0,0 +1,6 @@
+ ServerEvents.tags('item', event => {
+    for (var i = 0; i < colors.length; i++) {
```

Comment:
forEach? :D

Reply:
also no longer needed as written. Revamped to add `forge:terracotta/color` tags to each terracotta. Also did the same for glazed terracotta